pyproject.toml

```diff
@@ -56,11 +56,11 @@ requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning>=1.0.0,<2.0.0"]
 build-backend = "poetry_dynamic_versioning.backend"                          # object used to perform the build process
 
 [tool.poetry.dependencies] # must download to run
-python = ">=3.9"       # mandatory to declare the required python version
-matplotlib = ">=3.7.0" # Base requirement works, version specific to test outputs
-netcdf4 = "*"          # Base requirement
-numpy = "*"            # Base requirement
-pyresample = "*"       # Base requirement Geospatial image resampling
+python = ">=3.9"      # mandatory to declare the required python version
+matplotlib = "<3.9.0" # Base requirement works, version specific to test outputs
```

Comment:
@evrose54 Can you remind me why we're limiting this to `<3.9.0`? We're not currently doing that in `main`.

Reply:
I honestly forget why I had this here. I think it was due to the change in matplotlib which removed the functionality of ``matplotlib.cm.get_cmap`` and replaced it with ``matplotlib.pyplot.get_cmap``. I've updated portions of the code which used the old functionality; I'll test again and if that passes, I'll remove the updates I made to pyproject.toml

Reply:
This change has been removed. My comment above was correct.